doc/news/_preparation_next_release.md

```diff
@@ -157,7 +159,7 @@ The 0.9.\* series of Elektra is for development of Elektra 1.0.
 Elektra 1.0 will be incompatible to 0.8 and as such, we need a SO version bump.
```

Comment:
_This comment actually refers to line 146:_
Is there a reason why the the RPM instruction are in the DEB instructions block? They both also begin with `2.`.

Reply:
I tried to shorten it even more but I agree it might be a bit confusing. Note that it actually renders 2x "2.", so it is somewhat clear that only one of the second steps should be done. But please feel free to improve, simply push to this branch.

Reply:
I changed the installation guide a bit, I'm not sure if the additional headers are the right way, but it looks IMO a bit more structured. What do you think?

Reply:
I actually misunderstood which text belongs to DEB and what to RPM. Thank you for fixing it!